stdlib/public/core/Dictionary.swift

```diff
@@ -3697,6 +3701,7 @@ extension Dictionary._Variant {
       return Dictionary.Iterator(_native: native.makeIterator())
 #if _runtime(_ObjC)
     case .cocoa(let cocoa):
+      cocoaPath()
```

Comment:
This change allows the optimizer to eliminate the Cocoa path for Element types that aren't bridged verbatim. My guess is that this triggers the SILFunctionTransform crash -- I'll verify tomorrow.

Reply:
Ah, except the crash occurs on Linux. Hm.